helm-chart/renku/templates/data-service/deployment.yaml

```diff
@@ -142,5 +152,17 @@ spec:
         - name: server-options
           configMap:
             name: {{ template "renku.fullname" . }}-server-options
+        - name: encryption-key
+          secret:
+            secretName: {{ template "renku.fullname" . }}-secrets-storage
```

Comment:
Where is this defined? There does not seem to be any new definition for a secret with this name.